ui/src/containers/pipelines/EditPipeline.js

```diff
@@ -676,6 +678,20 @@ class EditPipeline extends Component {
     });
   }
 
+  updateInspectLimit(event) {
```

Comment:
(idk if this is the correct code for what i'm commenting)
This updates every character event. So if the field is empty and i want 100 records it will first filter to 1, then to 10 then to 100 and each time it filters it can take a few seconds to load.

I think it would be more user friendly if we updated on enter instead of events?

Reply:
Good catch, I agree! 👍

Reply:
Fixed with 2fc78b3